lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -320,6 +321,14 @@ PROCESS_THREAD(pbio_legodev_pup_process, ev, data) {
 
     PROCESS_BEGIN();
 
+    // Some hubs turn on power to the I/O ports in the bootloader. This causes
+    // UART sync delays after boot. It is faster to power cycle the I/O devices
+    // than it is to wait for long sync in most cases.
+    pbdrv_ioport_enable_vcc(false);
+    etimer_set(&timer, 500);
+    PROCESS_WAIT_EVENT_UNTIL(ev == PROCESS_EVENT_TIMER && etimer_expired(&timer));
+    pbdrv_ioport_enable_vcc(true);
+
```

Comment:
We could save code size on move hub and have faster start up on SPIKE hubs if we make this a configurable option and only enable it on the hubs that need it.